### DISCUSSION TITLE
제가 잘 한게 맞을까요?

### DISCUSSION THREAD

**Discussion by orlando-12** [changdoc/naver-vaccine-macro#232] [General]
현재 네이버 새로고침 중인 네이버 플레이스 지도 하나 
그리고 카카오 맵으로 메크로 설정 중인 브라우저 하나 이렇게 돌리고 있는데요!

방금 카카오 맵을 켜둔 브라우저가 개 킹받는 소리를 내면서 
네이버 예약 페이지로 넘어는 갔으나 예약엔 실패를 해서요

맞게한 방법일까요? 이대로 쭉 존버하면 되겠죠?

**Comment by changdoc** [comment]
네 예약 실패 창까지 진행이 되었다면 매크로 동작에 의해 자동으로 신청한 부분까진 정상 동작으로 볼 수 있습니다.
해당 병원에 동시에 신청이 몰려서 실패를 했을 가능성이 큰 듯 합니다.
